modules/signoz/README.md

```diff
@@ -0,0 +1,62 @@
+# Purpose
+The purpose of this module is to deploy the `Signoz` helm chart <https://github.com/SigNoz/charts/tree/main/charts/signoz>.
+
+SigNoz is an open-source APM. It helps developers monitor their applications 
+& troubleshoot problems, an open-source alternative to DataDog, NewRelic, etc. Open 
+source Application Performance Monitoring (APM) & Observability tool.
+
+
+## This module is a work in progress
```

Comment:
which is work in progress? the [signoz terraform module](https://registry.terraform.io/providers/SigNoz/signoz/latest/docs) itself?  or this specific setup of signoz?

Reply:
I noted the item below.

I am not using the terraform provider that you linked to, it does nit implement anything but a declarative configuration for some alerts.

The tool (spacelift) that I am using for Deploying the terraform scripts has a private module registry. That documentation is here:
https://docs.spacelift.io/vendors/terraform/module-registry


That private module registry would let us tag specific module versions that we create (Everything in the /modules directory of this project). It also lets us easily register modules from other git repositories, meaning we are not restricted to a mono repo style of code, we can use a mix.